coldfront/core/allocation/views_/secure_dir_views.py

```diff
@@ -757,3 +770,869 @@ def post(self, request, *args, **kwargs):
         return HttpResponseRedirect(
             reverse(f'secure-dir-manage-users-request-list',
                     kwargs={'action': self.action, 'status': 'pending'}))
+
+
+class SecureDirRequestWizard(LoginRequiredMixin,
```

Comment:
Putting this here--the "Manage" button becomes grayed out for a completed step. The admin may have made a mistake and so would need to change it back to pending.